inbox/Makefile

```diff
@@ -0,0 +1,8 @@
+OUTDIR?=../build/
+
+xeps=$(wildcard *.xml)
+
+.PHONY: all
+all: html
+
+html: $(patsubst %.xml, $(OUTDIR)/%.html, $(xeps))
```

Comment:
This doesn't appear to actually do anything; we probably shouldn't copy/paste the logic into the inbox anyways, I think building inbox files can be done from the main makefile too with some tweaks.